pkg/skaffold/parser/config.go

```diff
@@ -129,6 +129,34 @@ func getConfigs(cfgOpts configOpts, opts config.SkaffoldOptions, r *record) (Ska
 		seen[cfgName] = true
 	}
 
+	// validate that manifest files referenced in config exist
+	for _, cfg := range parsed {
+		// TODO(aaron-prindle) fix nil pointer dereference
+		if cfg.(*latestV1.SkaffoldConfig).Deploy.KubectlDeploy == nil {
+			break
+		}
+		manifests := cfg.(*latestV1.SkaffoldConfig).Deploy.KubectlDeploy.Manifests
+		// TODO(aaron-prindle) check for other manifest types - helm, kpt?
+		for _, pattern := range manifests {
+			// TODO(aaron-prindle) verify this is only run when manifests are local (vs remote)
+			if util.IsURL(pattern) {
+				continue
+			}
+			// TODO(aaron-prindle) verify this is the correct way to get the dir
+			cwd, _ := util.RealWorkDir()
+			expanded, err := filepath.Glob(filepath.Join(cwd, pattern))
+			if err != nil {
+				return nil, err
+			}
+			if len(expanded) == 0 {
+				return nil, fmt.Errorf("pattern %q did not match any file", pattern)
```

Comment:
should this return `ConfigMissingManifestFile`? or `ConfigHasMissingManifestFileErr`

Reply:
This was still WIP at the time, you are correct it should return `ConfigHasMissingManifestFileErr`.  This now returns `ConfigHasMissingManifestFileErr`